### DISCUSSION TITLE
Netbox Installation issue

### DISCUSSION THREAD

**Discussion by vdodin** [netbox-community/netbox#16036] [Help Wanted!] [answered]
Hi, 

I'm trying to install Netbox on Ubuntu 22.04.4 LTS

Followed the official procedure https://github.com/netbox-community/netbox/tree/develop/docs/installation 

Got to section 3-Netbox.md and setup all the necessary parameters. 

Ran the  sudo /opt/netbox/upgrade.sh

It's completed with: 

Removing expired user sessions (python3 netbox/manage.py clearsessions)...
--------------------------------------------------------------------
WARNING: No existing virtual environment was detected. A new one has
been created. Update your systemd service files to reflect the new
Python and gunicorn executables. (If this is a new installation,
this warning can be ignored.)

netbox.service ExecStart:
  /opt/netbox/venv/bin/gunicorn

netbox-rq.service ExecStart:
  /opt/netbox/venv/bin/python

After modifying these files, reload the systemctl daemon:
  > systemctl daemon-reload

Trying to open the web with http://IP_ADRESS:8000, the page doesn't come up.
Under configuration.py, in the section of ALLOWED_HOSTS = ['the_IP_Of_THE_Server']. 

Did I miss anything? Or do I need to do something with netbox services? (They seems to be not running at all)

Thanks

Vlad

**Comment by julianstolp** [comment, marked as answer]
I'm not sure which step you're stuck on. In step 3 the server is started by the command `python3 manage.py runserver 0.0.0.0:8000 --insecure`.
The services only come in step 4 and then the HTTP server, then NetBox can only be accessed via port 80 or 443.
I suggest you take a look at the documentation website and not the raw markdown version in GitHub https://docs.netbox.dev/en/stable/installation/3-netbox/

**Comment by vdodin** [comment]
Thank you @julianstolp,
I've had to run source opt/netbox/venv/bin/activate at first
Then went to /opt/netbox/netbox$ 
And finally ran the command: python3 manage.py runserver [server_Ip]:8000 --insecure
Then, I've got into my web page with the login. It looks good. Now, if I leave the session open it runs, once the session been idled, no page access. I guess there are more things to configure in the next steps.
